ios/ImagePickerUtils.m

```diff
@@ -156,4 +156,18 @@ + (UIImage*)resizeImage:(UIImage*)image maxWidth:(float)maxWidth maxHeight:(floa
     return newImage;
 }
 
++ (PHAsset *)fetchPHAsset:(NSDictionary<NSString *,id> *)info
```

Comment:
I may be wrong, but is this method only for iOS 13 and below? As iOS 14 uses the PHPhotoPicker callback, if so shall we rename this to something like `fetchPHAssetFromAssetURL` or `fetchPHAssetOnOS13` ? Lmk!

Reply:
Sounds good! Let's update that, then I'll merge 🎉 Thanks for the contributions !

Reply:
@lukebrandonfarrell updated 🙌